ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -42,6 +42,9 @@ export default class TestDataModal extends React.Component {
       filterText: '',
       loading: true,
       selectedUnits: new Set(),
+      defaultTag: 'all',
```

Comment:
I noticed the change. But it could be better.
Because it's being used as a temporary private variable, you have to do some black magic to get it back in `getTagOptions()`.

Lookup for Javascript's new static properties & leverage them.